org.eclipse.jdt.core/model/org/eclipse/jdt/internal/core/CompilationUnit.java

```diff
@@ -327,6 +327,11 @@ public char[] getContents() {
 		public CompilationUnit originalFromClone() {
 			return CompilationUnit.this;
 		}
+		@Override
```

Comment:
Can this be submitted upstream?

Reply:
Was submitted here:  https://github.com/eclipse-jdt/eclipse.jdt.core/pull/3216